PostController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+
+    public function postAction() {
+
+        //avoid notices warnings
+        !isset($_POST['livechat_license_number']) ? $livechat_license_number =  '': $livechat_license_number = $_POST['livechat_license_number'];
+        !isset($_POST['livechat_groups']) ? $livechat_groups =  '0' :  $livechat_groups =  $_POST['livechat_groups'];
+        !isset($_POST['livechat_params']) ? $livechat_params =  '' :  $livechat_params =  $_POST['livechat_params'];
```

Comment:
Use Magento syntax to get the post data , instead of use php post method.
Also need to correct the variable naming convention.
Eg :
$postData = $this->getRequest()->getPost();
$livechatParams = isset($postData['livechat_params']) ? $postData['livechat_params'] : "";